docs/data/routes/docs/nextjs/page-routing/error-routes/en.md

```diff
@@ -3,4 +3,22 @@ name: error-routes
 routeTemplate: ./data/component-templates/article.yml
 title: Error Routes
 ---
-# Error Routes
\ No newline at end of file
+# Error Routes
```

Comment:
Should we make this "Error Pages" instead (follows our names below and Next.js naming)? (here and in our nav)

Reply:
Agree, changed to "Error Pages"